classes/utils/Biome.py

```diff
@@ -0,0 +1,95 @@
+        self.id = id
+        self.string = string
+
+    def id_to_biome(id):
+        for biome in Biome:
+            if biome.id == id:
+                return biome
+        return None
+
+    def name_to_biome(name):
+        for biome in Biome:
+            if biome.name == name:
+                return biome
+        return None
+
+    def string_to_biome(string):
+        for biome in Biome:
+            if biome.biome == string:
+                return biome
+        return None
+
+    INVALID_BIOME = (-1, 'Invalid Biome')
+    OCEAN = (0, 'Ocean')
+    PLAINS = (1, 'Plain')
+    DESERT = (2, 'Desert')
+    MONTAINS = (3, 'Montains')
+    EXTREME_HILLS = (3, 'ExtremeHills')
+    FOREST = (4, 'Forest')
+    TAIGA = (5, 'Taiga')
+    SWAMPLAND = (6, 'Swampland')
+    RIVER = (7, 'River')
+    HELL = (8, 'Hell')
+    NETHER = (8, 'Nether')
+    SKY = (9, 'Sky')
+    END = (9, 'End')
+    FROZEN_OCEAN = (10, 'FrozenOcean')
+    FROZEN_RIVER = (11, 'FrozenRiver')
+    ICE_PLAINS = (12, 'IcePlains')
+    TUNDRA = (12, 'Tundra')
+    ICE_MOUNTAINS = (13, 'IceMountains')
+    MUSHROOM_ISLAND = (14, 'MushroomIsland')
+    MUSHROOM_SHORE = (15, 'MushroomShore')
+    BEACH = (16, 'Beach')
+    DESERT_HILLS = (17, 'DesertHills')
+    FOREST_HILLS = (18, 'ForestHills')
+    TAIGA_HILLS = (19, 'TaigaHills')
+    EXTREME_HILLS_EDGE = (20, 'ExtremeHillsEdge')
+    JUNGLE = (21, 'Jungle')
+    JUNGLE_HILLS = (22, 'JungleHills')
+    JUNGLE_EDGE = (23, 'JungleEdge')
+    DEEP_OCEAN = (24, 'DeepOcean')
+    STONE_BEACH = (25, 'StoneBeach')
+    COLD_BEACH = (26, 'ColdBeach')
+    BIRCH_FOREST = (27, 'BirchForest')
+    BIRCH_FOREST_HILLS = (28, 'BirchForestHills')
+    ROOFED_FOREST = (29, 'RoofedForest')
+    COLD_TAIGA = (30, 'ColdTaiga')
+    COLD_TAIGA_HILLS = (31, 'ColdTaigaHills')
+    MEGA_TAIGA = (32, 'MegaTaiga')
+    MEGA_TAIGA_HILLS = (33, 'MegaTaigaHills')
+    EXTREME_HILLS_PLUS = (34, 'ExtremeHillsPlus')
+    SAVANNA = (35, 'Savanna')
+    SAVANNA_PLATEAU = (36, 'SavannaPlateau')
+    MESA = (37, 'Mesa')
+    MESA_PLATEAU_F = (38, 'MesaPlateauF')
+    MESA_PLATEAU = (39, 'MesaPlateau')
+
+    # Variant
+    SUNFLOWER_PLAINS = (129, 'SunflowerPlains')
+    DESERT_M = (130, 'DesertM')
+    EXTREME_HILLS_M = (131, 'ExtremeHillsM')
+    FLOWER_FOREST = (132, 'FlowerForest')
+    TAIGA_M = (133, 'TaigaM')
+    SWAMPLAND_M = (134, 'SwamplandM')
+    ICE_PLAINS_SPIKES = (140, 'IcePlainsSpikes')
+    JUNGLE_M = (149, 'JungleM')
+    JUNGLE_EDGE_M = (151, 'JungleEdgeM')
+    BIRCH_FOREST_M = (155, 'BirchForestM')
+    BIRCH_FOREST_HILLS_M = (156, 'BirchForestHillsM')
+    ROOFED_FOREST_M = (157, 'RoofedForestM')
+    COLD_TAIGA_M = (158, 'ColdTaigaM')
+    MEGA_SPRUCE_TAIGA = (160, 'MegaSpruceTaiga')
+    MEGA_SPRUCE_TAIGA_HILLS = (161, 'MegaSpruceTaigaHills')
+    EXTREME_HILLS_PLUS_M = (162, 'ExtremeHillsPlusM')
+    SAVANNA_M = (163, 'SavannaM')
+    SAVANNA_PLATEAU_M = (164, 'SavannaPlateauM')
+    MESA_BRYCE = (165, 'MesaBryce')
```

Comment:
I'd use Eroded Badlands for the enum name, however the namespaced ID is correct (even though I thought it doesn't exist at first)

Reply:
I'll use both MESA_BRYCE and ERODED_BADLANDS

Reply:
that may actually have side effects (like in comparision). I'd use ERODED_BADLANDS over MESA_BRYCE  as MESA_BRYCE  is almost never used pretty much anywhere